tests/robustness/validate/watch.go

```diff
@@ -32,13 +33,10 @@ func validateWatch(t *testing.T, cfg Config, reports []traffic.ClientReport) []m
 		// TODO: Validate Resumable
 		validateBookmarkable(t, r)
 	}
-	validateEventsMatch(t, reports)
-	// Expects that longest history encompasses all events.
-	// TODO: Use combined events from all histories instead of the longest history.
-	eventHistory := longestEventHistory(reports)
+	eventHistory := mergeWatchEventHistory(t, reports)
 	// TODO: Validate that each watch report is reliable, not only the longest one.
```

Comment:
Is this todo still valid now we are removing `func longestEventHistory` and merging histories?

Reply:
Yes, we still need to improve validateReliable.